modules/ROOT/pages/graph-analytics/index.adoc

```diff
@@ -37,12 +37,12 @@ With the Graph Analytics plugin enabled:
 * If you use Python, start with the link:{neo4j-docs-base-uri}/graph-data-science-client/current/tutorials/tutorials/[Python client tutorials].
 
 [[aura-gds-serverless]]
-== Graph Analytics Serverless
+== Aura Graph Analytics
 
-Graph Analytics Serverless allows you to use the Graph Data Science library regardless of where your source data is stored.
+Aura Graph Analytics allows you to use the Graph Data Science library regardless of where your source data is stored.
 It runs in Aura as a link:{gds-sessions-page}[dedicated service] optimised for analytics workloads, with no memory or compute resources shared with your data store.
 
-You can enable, disable, and configure Graph Analytics Serverless on the organization level in the xref:visual-tour/index.adoc#_settings_for_graph_analytics_serverless[organization settings].
+You can enable, disable, and configure Aura Graph Analytics serverless on the organization level in the xref:visual-tour/index.adoc#_settings_for_graph_analytics_serverless[organization settings].
```

Comment:
```suggestion
You can enable, disable, and configure Aura Graph Analytics on the organization level in the xref:visual-tour/index.adoc#_settings_for_graph_analytics_serverless[organization settings].
```

I'd just get rid of `serverless` everywhere, as it does not add anything.

Reply:
If we have to, maybe let's use parentheses (`Aura Graph Analytics (serverless)`) at least? Because otherwise it seems part of either the product name or of the instructions ("Enable, disable, and configure _something_ serverless").

Anyway, I would still remove it from this section, which is only related to Aura Graph Analytics anyway. We could use the serverless hint in the comparison section only.